fixed broken link (#1311)
OfficeDev/office-js-docs-pr: docs/develop/authorize-to-microsoft-graph-without-sso.md

For more information about recommended libraries for working with Microsoft Identity Platform (formerly AAD v.2.0), see [Microsoft identity platform authentication libraries](/azure/active-directory/develop/reference-v2-libraries).

> For more information how to get Microsoft Graph data from an Office Add-in, see [Get started with Microsoft Graph and ASP.NET](https://developer.microsoft.com/en-us/graph/get-started/asp.net).

Reviewer: ```suggestion
The following samples get Microsoft Graph data from an Office Add-in:

- [Office Add-in Microsoft Graph ASP.NET](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Office-Add-in-Microsoft-Graph-ASPNET)
- [Outlook Add-in Microsoft Graph ASP.NET](https://github.com/OfficeDev/PnP-OfficeAddins/tree/master/Samples/auth/Outlook-Add-in-Microsoft-Graph-ASPNET)
```
